css/style.css

```
@@ -0,0 +1,110 @@
+*{
+	margin: 0;
+	padding: 0;
+	font-family: sans-serif;
+    text-align: center;
+    box-sizing: border-box;
+}
```

I recommend that you use a more precise selector or class to center a text because you don't know about the future were you might want a text to be aligned to the left.
so for example try using `h1` instead of `*`.


```suggestion
*{
	margin: 0;
	padding: 0;
	font-family: sans-serif;
    box-sizing: border-box;
}
h1 {
    text-align: center;
}
```